src/moduler/aktivitet/visning/avtalt-container/SendtSuksesInfoLinje.tsx

```diff
@@ -0,0 +1,88 @@
+import { AlertStripeSuksess } from 'nav-frontend-alertstriper';
+import { Normaltekst } from 'nav-frontend-typografi';
+import React from 'react';
+
+import LenkeTilDialog from '../../../dialog/DialogLink';
+import DeleLinje from '../delelinje/delelinje';
+import { IKKE_SEND_FORHAANDSORIENTERING, SEND_FORHAANDSORIENTERING, SEND_PARAGRAF_11_9 } from './AvtaltForm';
+
+interface Props {
+    erManuellKrrKvpBruker: boolean;
+    forhaandsorienteringstype: string;
+    mindrennSyvDager: boolean;
+    dialogId?: string;
+    className?: string;
+    skalBrukeNyForaandsorientering: boolean;
+}
+
+export const SendtSuksesInfoLinje = (props: Props) => {
+    const {
+        erManuellKrrKvpBruker,
+        forhaandsorienteringstype,
+        mindrennSyvDager,
+        dialogId,
+        className,
+        skalBrukeNyForaandsorientering,
+    } = props;
+    const tekst = getTekst(erManuellKrrKvpBruker, forhaandsorienteringstype, mindrennSyvDager);
+
+    if (!skalBrukeNyForaandsorientering) {
+        return <Gammel tekst={tekst} dialogId={dialogId} className={className} />;
+    }
+
+    return (
+        <div>
```

Comment:
Kan denne kanskje erstattes med dette
```suggestion
        <>
```